apps/frontend/src/app/components/header/header.component.html

```diff
@@ -24,9 +24,17 @@ <h1 class="title" routerLink="/">Verseghy</h1>
         <a (click)="drawer = !drawer" href="" routerLink="/misc/authors">Fejlesztők</a>
       </div>
     </div>
-    <a class="ekreta" href="https://klik035992001.e-kreta.hu/Adminisztracio/Login" target="_blank">
-      <img alt="" src="assets/icons/kreta2.png" />
-    </a>
+    <div (click)="kretaSubmenu = !kretaSubmenu" [ngClass]="{ open: kretaSubmenu }" class="menu ekreta">
+      <span><img alt="" src="assets/icons/kreta2.png"/></span> <i class="material-icons arrow">expand_more</i>
```

Comment:
![Screenshot from 2019-05-24 00-09-43](https://user-images.githubusercontent.com/14909670/58289957-4fee9100-7db8-11e9-9e07-9b775598d253.png)

As you can see on the picture, the KRÉTA menu is way taller than the other header items, causing it to flow out of the header. Please fix it so it has the same height as the other header items

Reply:
Fixed in 2239f82
Removing `<span>` tag fixed it